packages/replayio/src/commands/record.ts

```diff
@@ -176,3 +171,57 @@ async function record(url: string = "about:blank") {
 
   await exitProcess(0);
 }
+
+async function reportRecorderCrash(stderr: string) {
+  console.log("\nSomething went wrong while recording. Try again.");
+
+  const errorLogPath = getReplayPath("recorder-crash.log");
+  writeFileSync(errorLogPath, stderr, "utf8");
+  console.log(dim(`More information can be found in ${errorLogPath}`));
+
+  const accessToken = await checkAuthentication();
+
+  if (!accessToken) {
+    return;
+  }
+
+  const userAgent = getUserAgent();
+
+  const formData = new FormData();
+
+  formData.set("buildId", getCurrentRuntimeMetadata("chromium")?.buildId ?? "unknown");
+  formData.set("createdAt", new Date().toISOString());
+  formData.set("userAgent", userAgent);
+
+  formData.append(
+    "log",
+    new File([await readFile(errorLogPath)], "log.txt", { type: "text/plain" })
+  );
+
+  const latestCrashpad =
+    runtimeMetadata.crashpadDirectory &&
+    (await findMostRecentFile(runtimeMetadata.crashpadDirectory, fileName =>
+      fileName.endsWith(".dmp")
```

Comment:
this directory really should only have those `.dmp` files, I'm using this predicate just as an extra precaution